.github/workflows/ci.yml

```diff
@@ -122,12 +122,14 @@ jobs:
 
   tutorial-e2e:
     needs: check
+
     strategy:
       matrix:
-        os: [ubuntu-latest]
-      fail-fast: true
-    name: 🌲 Tutorial E2E / ${{ matrix.os }} / node 18 latest
-    runs-on: ${{ matrix.os }}
+        bundler: [vite, webpack]
+
+    name: 🌲 Tutorial E2E / ${{ matrix.bundler }} / node 18 latest
```

Comment:
I've included the bundler in the check's name so I'll have to change the branch protection rules to accommodate